README.md

```diff
@@ -15,3 +15,24 @@ Paste following line in powershell in administrator mode. Search for powershell
 PowerShell -ExecutionPolicy Bypass -Command "& {Invoke-Expression (Invoke-WebRequest -Uri 'https://raw.githubusercontent.com/dtudk/pythonsupport-scripts/main/Windows_AutoInstall.ps1' -UseBasicParsing).Content}"
 ```
 
+# Pilot: Installing dependancies for converting notebooks to pdf in Vs Code 
```

Comment:
typo, `dependencies`

Reply:
also `VS Code`

Reply:
This was changed in the newest commit. Can you verify and allow merging?